### ISSUE TITLE
Different MSV and EMAC values

### ISSUE THREAD

**Issue by Ayubirad** [twmacro/pyyeti#12]
Dear @twmacro

Another minor issue that I've encountered is the MSV and EMAC values from Auto-selected modes fit. As you can see, they differ from the values obtained earlier in Current fit. Why is that?

```
Current fit includes all modes:
  Mode   Freq. (Hz)     Zeta        MSV      EMAC       MPC       MAC
  --------------------------------------------------------------------
     1     1.47868     1.67646    *1.000     0.211    *1.000    *1.000
     2     0.04590     0.22195    *0.705    *0.758    *0.961    *1.000
     3     0.82994     0.20514     0.196     0.005    *0.780    *0.999
     4     1.57808     0.04575     0.099     0.232     0.414    *0.986
     5     1.77248     0.62986    *0.501     0.000     0.375    *1.000
     6     2.24217    -0.00271     0.308    *0.934    *0.956    *1.000
     7     2.61220    -0.00286     0.187    *0.743     0.317    *1.000
     8     3.22967     0.05908     0.204     0.000     0.633    *0.999
     9     3.64498     0.03549     0.319     0.316     0.643    *1.000
    10     4.09857     0.04775    *0.562     0.126    *0.947    *1.000
*   11     4.38722     0.02742    *0.759    *0.857    *0.994    *1.000
    12     4.80675     0.32866    *0.594     0.002     0.592    *1.000
    13     4.84957     0.02361     0.331     0.251    *0.830    *1.000
    14     5.12087    -0.00059     0.304    *0.656    *0.910    *1.000
    15     6.24316     0.00172     0.113     0.121     0.270    *0.987
    16     6.75059     0.30834    *0.637     0.219     0.465    *1.000
    17     8.04582     0.03026     0.123     0.001     0.153     0.939
    18     8.99466     0.13668    *0.351     0.000     0.261    *0.999
    19     9.20704     0.00521     0.242    *0.557     0.370    *0.999
    20     9.45505     0.01205     0.231    *0.708     0.084    *0.999
    21     9.86625     0.00762     0.295     0.380     0.304    *1.000
    22    10.75880     0.00467     0.296    *0.947     0.250    *1.000
    23    11.36226     0.05542    *0.368     0.000     0.277    *0.999
    24    11.88720     0.03577    *0.389     0.000     0.436    *0.999
*   25    12.24301     0.00387    *0.687    *0.913    *0.751    *1.000
    26    12.83397     0.28727    *0.860     0.000     0.205    *1.000
*   27    12.87371     0.00361    *0.366    *0.826    *0.806    *1.000
    28    13.06253     0.07408    *0.666     0.000     0.133    *1.000
    29    13.42139     0.01470     0.224     0.003     0.509    *0.998
    30    13.62301     0.05796    *0.519     0.000     0.195    *1.000
    31    14.56481     0.00409     0.225    *0.512     0.692    *0.999
    32    15.36630     0.00618     0.191     0.023     0.052    *0.999
    33    15.80336     0.03440    *0.457     0.004     0.018    *1.000
    34    15.92065     0.01566    *0.397     0.464     0.138    *1.000
    35    16.54866     0.00636     0.273    *0.593     0.687    *1.000
    36    17.01757     0.05340    *0.642     0.000     0.268    *1.000
    37    17.26609     0.01108    *0.614    *0.826     0.420    *1.000
    38    17.58486     0.00846    *0.625     0.363    *0.887    *1.000
    39    17.97310     0.02207    *0.410     0.000     0.505    *1.000
*   40    19.11882     0.00575    *0.691    *0.653    *0.859    *1.000
    41    19.16177     0.06449    *0.583     0.000     0.028    *1.000
    42    19.48823     0.00933    *0.418     0.000    *0.779    *1.000
    43    19.89872     0.01690     0.322     0.000     0.397    *0.998
    44    20.91029     0.03972    *0.475     0.000     0.314    *1.000
    45    20.91909     0.01238     0.249     0.030     0.068    *0.999
    46    21.73313     0.01034     0.267     0.000     0.618    *0.998

Auto-selected modes fit:
  Mode   Freq. (Hz)     Zeta        MSV      EMAC       MPC       MAC
  --------------------------------------------------------------------
     1     4.38722     0.02742     1.000     0.941     0.994     1.000
     2    12.24301     0.00387     0.904     0.833     0.751     1.000
     3    12.87371     0.00361     0.482     0.702     0.806     1.000
     4    19.11882     0.00575     0.910     0.925     0.859     1.000
```

Best regards,
Ayubirad

**Comment by twmacro** [comment]
Hi @Ayubirad,

That's a great question! :-) I'll take a look at why that happens and report back ... I hope within a day or two. (I just got back from vacation and my brain needs to get reoriented.)

Thank you!!

Best regards,
Tim

**Comment by twmacro** [comment]
Fixed in 1.2.7.
